collect_app/src/main/java/org/odk/collect/android/activities/CollectAbstractActivity.java

```diff
@@ -47,7 +45,6 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         DaggerUtils.getComponent(this).inject(this);
 
         themeUtils = new ThemeUtils(this);
```

Comment:
This seems to be unused now so you can remove it.

Reply:
Huh good catch! I thought it was still being used.